src/ui/Form/Form.stories.tsx

```diff
@@ -0,0 +1,45 @@
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+
+import { Form as FormComponent } from './Form';
+
+export default {
+  title: 'Design System/Components/Form',
+  component: FormComponent,
+  argTypes: {
+    children: {
+      table: {
+        disable: true,
+      },
+    },
+    css: {
+      table: {
+        disable: true,
+      },
+    },
+    content: {
+      table: {
+        disable: true,
+      },
+    },
```

Comment:
`content` is not a prop for the `Form`, you should disable `as` and `ref` instead

Reply:
what is 'as'

Reply:
> what is 'as'

It's just a prop